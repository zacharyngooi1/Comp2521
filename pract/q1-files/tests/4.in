 
7, 9, 14, 55, 82

