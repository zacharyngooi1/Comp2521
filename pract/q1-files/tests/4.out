list1: 
list2: 7, 9, 14, 55, 82
answer List: 7, 9, 14, 55, 82
